### PR TITLE
fixed config problem when empty values are given

### LINKED ISSUE #50: If web.auth.mainTeam.config is empty helm chart errors out
Therefore the default install as described in the README is not working. 

Reproduce:
Follow README for basic install.

Reason: 
no default value of ifs around last 2 lines in tempates/web-configmap.yml.

### PR DESCRIPTION
**Pull request by d-likes-it**
a default empty string is inserted into the files (that are not used in the deployment in the default case)

Fixes #50.

### CODE REVIEW

**Comment by cirocosta** [comment]
Thanks for the PR!

Would you mind sharing under which version you got the problem?  And the error that's reported too? I wasn't able to reproduce here on `1.17.0`:

```console
$ helm install --namespace test --name test .
...

$ kubectl -n test get configmap test-web -o yaml
apiVersion: v1
data:
  config-rbac.yml: ""
  main-team.yml: ""
kind: ConfigMap
...
```

Regardless,  I think it's a change that makes sense - we set defaults for the fields on the secrets created by the chart, although with a slightly different syntax:

https://github.com/concourse/concourse-chart/blob/b497dc9cabac7ba5aec786e77bc08ae3abee273f/templates/web-secrets.yaml#L21-L23

To get the PR ready, we'd need one more thing: getting the version https://github.com/concourse/concourse-chart/blob/b497dc9cabac7ba5aec786e77bc08ae3abee273f/Chart.yaml#L3

bumped there (patch bump).

wdyt @taylorsilva ?

thanks!

File: Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: concourse
-version: 9.0.0
+version: 9.0.1
```
[the remark this answers is not in the document]
Comment (cirocosta) [reply Chart.yaml]: oooooh interesting, thanks for letting me know, @taylorsilva !

sorry @d-likes-it 😅

**Comment by cirocosta** [comment]
Hey @d-likes-it , would you mind updating `Chart.yaml` without the version bump that I had suggested (see @taylorsilva 's message).

btw, I was able to reproduce on 1.17 (just noticed that you had put the version of the kubernetes cluster there in the original issue).

thanks!

**Comment by taylorsilva** [comment]
Forgot about this. It got fixed in #67 
Thanks for trying to fix this though!

**Pull request closed without merging**